core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -65,14 +63,12 @@
       </a>
     </li>
 
-    {% if username %}
-      <li ng-if="isLargeScreen" ng-class="{'active': getTabStatuses().active === 'stats'}">
-        <a href="#" tooltip="Statistics" tooltip-placement="bottom" ng-click="selectStatsTab()"
-           class="oppia-editor-navbar-tab-anchor">
-          <i class="material-icons">&#xE24B;</i>
-        </a>
-      </li>
-    {% endif %}
+    <li ng-if="isLargeScreen" ng-class="{'active': getTabStatuses().active === 'stats'}" ng-if="username">
```

Comment:
Please combine the two ng-ifs. Btw, I recommend generally putting ng-ifs at the front of the attr list, too.